modules/updating-eus-to-eus-upgrade-console.adoc

```diff
@@ -51,4 +51,8 @@ If pools are paused, the cluster is not permitted to upgrade to any future minor
 +
 You can verify that your pools have updated on the *MachineConfigPools* tab under the *Compute* page by confirming that the *Update status* has a value of *Up to date*.
 +
+[IMPORTANT]
+====
+When you update a cluster that contains Red Hat Enterprise Linux (RHEL) worker machines, those workers temporarily become unavailable during the update process. You must run the upgrade playbook against each RHEL machine as it enters the NotReady state for the cluster to finish updating.
```

Comment:
Just a couple style guide type things here to change:

* we use variables that become `Red Hat Enterprise Linux (RHEL)` and `RHEL` in the output
* in almost all cases, use "compute" instead of "worker"
* backticks around a literal status like `NotReady`

```suggestion
When you update a cluster that contains {op-system-base-full} compute machines, those machines temporarily become unavailable during the update process. You must run the upgrade playbook against each {op-system-base} machine as it enters the `NotReady` state for the cluster to finish updating.
```